src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAnAppWithoutTransitiveProjectRefs.cs

```diff
@@ -18,67 +23,148 @@ public GivenThatWeWantToBuildAnAppWithoutTransitiveProjectRefs(ITestOutputHelper
         {
         }
 
-        [Fact]
+        [WindowsOnlyFact]
```

Comment:
Why did this become windows only?

Reply:
It fails on crossplat due to missing reference assemblies to the referenced full frameworks: 472, 462, and 461

Reply:
So we can actually use Microsoft.NETFramework.ReferenceAssemblies to test these x-plat and we do when you use the TestProject API to create projects on the fly:

https://github.com/dotnet/sdk/blob/ff80c824a9e62ec721b467c9f6b0b407fbb14fa0/src/Tests/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs#L183-L192

cc @dsplaisted

Reply:
For checked in test projects, you'd have to add the package ref.

Reply:
Incidentally, we should probably use the shipping preview on nuget.org of Microsoft.NETFramework.ReferenceAssemblies now.

Reply:
I'll bump it up in a separate PR